src/main/java/step3/RacingGame.java

```diff
@@ -0,0 +1,86 @@
+package step3;
+
+import java.io.IOException;
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class RacingGame {
+
+	private static final int BOUNDARY_NUMBER = 4;
+
+	private static final int MAX_NUMBER = 10;
+
+	private int turnCount;
+
+	private List<Car> participants;
+
+	private List<String> winners;
+
+	public RacingGame() {
+		this.participants = new ArrayList<>();
+	}
+
+	public static void main(String[] args) throws IOException {
+		RacingGame racingGame = new RacingGame();
+		racingGame.start();
+	}
+
+	private void start() throws IOException {
+		this.participants = registParticipants(InputView.getParticipantNames());
+		this.turnCount = InputView.getTurnCount();
+		int remainTurn =  doRacing();
+	}
+
+	List<Car> registParticipants(String[] participantNames) {
```

Comment:
메서드 명은 동사 혹은 동사 + 명사 로 하는 경우가 많습니다 :)
```suggestion
	List<Car> registerParticipants(String[] participantNames) {
```